Final fixes (#232)
Adyen/adyen-react-native: docs/Configuration.md

- **amount** - Amount to be displayed on the Pay Button. It expects an object with the minor units value and currency properties. For example, `{ value: 1000, currency: 'USD' }` is **10$**. For card pre-authorisation set amount to `0` (zero).
- **countryCode** - The shopper's country code in ISO 3166-1 alpha-2 format. Example: `"NL"` or `"US"`.
- **shopperLocale** - 🚧 Work in progress. In current version this property only localises payment methods names. Default OS's locale is used for localisation.
- **returnUrl** - For iOS, this is the URL to your app, where the shopper should return, after a redirection. Maximum of 1024 characters. For more information on setting a custom URL scheme for your app, read the [Apple Developer documentation](https://developer.apple.com/documentation/uikit/inter-process_communication/allowing_apps_and_websites_to_link_to_your_content/defining_a_custom_url_scheme_for_your_app).
  For Android, this value is automatically overridden by `AdyenCheckout`.

> ⚠️ To show the amount on the **Pay** button both _amount_ and _countryCode_ must be set.

## Analytics

> - **enabled** - Enable/Disable all analytics. Defaults to `true`. ⚠️ This feature only availalbe fron v68 of Adyen API

Reviewer: ```suggestion
- **enabled** - Enable/Disable all analytics. Defaults to `true`. ⚠️ This feature only available from v68 of Adyen API
```

- **payload** - Data to be sent along with the event data

## React Native SDK provides following configurations for components:

### DropIn component

> - **showPreselectedStoredPaymentMethod** - Determines whether to enable preselected stored payment method view step. Defaults to `true`.

Reviewer: ```suggestion
- **showPreselectedStoredPaymentMethod** - Determines whether to enable the preselected stored payment method view step. Defaults to `true`.
```

- **skipListWhenSinglePaymentMethod** - Determines whether to enable skipping payment list step when there is only one non-instant payment method. Defaults to `false`.

### Card component

> - **showStorePaymentField** - Indicates if the field for storing the card payment method should be displayed in the form. Defaults to true.

Reviewer: ```suggestion
- **showStorePaymentField** - Indicates if the field for storing the card payment method should be displayed in the form. Defaults to `true`.
```

> - **holderNameRequired** - Indicates if the field for entering the holder name should be displayed in the form. Defaults to false.

Reviewer: ```suggestion
- **holderNameRequired** - Indicates if the field for entering the holder name should be displayed in the form. Defaults to `false`.
```

- **hideCvcStoredCard** - Indicates whether to show the security code field on a stored card payment. Defaults to false.
- **hideCvc** - Indicates whether to show the security code field at all. Defaults to false.
- **addressVisibility** - Indicates the display mode of the billing address form. Options: `"none"`, `"postal"`, `"full"`. Defaults to `"none"`.
- **kcpVisibility** - Indicates whether to show the security fields for South Korea issued cards. Options: `"show"` or `"hide"`. Defaults to `"hide"`.
- **socialSecurity** - Indicates the visibility mode for the social security number field (CPF/CNPJ) for Brazilian cards. Options: "show" or `"hide"`. Defaults to `"hide"`.
- **supported** - The list of allowed card types. By default uses list of `brands` from payment method. Fallbacks to list of all known cards.

### ApplePay component

> - **merchantID** - The merchant identifier for apple pay.

Reviewer: ```suggestion
- **merchantID** - The merchant identifier for Apple Pay.
```

Author: Fixed in [7567c07](https://github.com/Adyen/adyen-react-native/pull/232/commits/7567c071a9dbab954f3c478a2550fcc483717ef4)
